src/components/structures/TabbedView.tsx

```diff
@@ -118,13 +119,24 @@ export default class TabbedView extends React.Component<IProps, IState> {
     public render(): React.ReactNode {
         const labels = this.props.tabs.map(tab => this._renderTabLabel(tab));
         const panel = this._renderTabPanel(this.props.tabs[this._getActiveTabIndex()]);
-
+        
         return (
             <div className="mx_TabbedView">
                 <div className="mx_TabbedView_tabLabels">
                     {labels}
                 </div>
                 {panel}
+                {customConfig.tabbedViewdisplaySecondaryLogo === true &&
+                    <div className="mx_AuthFooter_TabbedView">
+                        <a href="" target="_blank" rel="noreferrer noopener">                        
+                        </a>
+                        <span className="mx_AuthFooter_brand_TabbedView">
+                            <img className="mx_AuthFooter_Image_Tabbed" src="themes/element/img/logos/ACTGov_inline_rev_black.png" alt="ACT Govt Logo"/>
```

Comment:
The image path and ACT Govt Logo need to be externalised into the config too pls